### ISSUE TITLE
[Comprehension] User doesn't have enough information to complete a task.

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#77115]
_This ticket was generated during the pilot process for the new VA.gov experience standards. Please tag `@platform-governance-team-members` on Slack if you need assistance interpreting this ticket._

## Product Information
**Team:** Accredited Representation Management Team
**Product:** Find a Representative (MVP)
**Feature:** Find a Representative

## Findings details
**VA.gov Experience Standard - issue:** User doesn't have enough information to complete a task.
**VA.gov Experience Standard - category:** Comprehension
**Launch-blocking:** No
**Collab Cycle Reviewer:** @briandeconinck (Accessibility)

### Description

Screen reader users often navigate a page by heading, pulling up a list of all the headings and then jumping down to the one they're interested in. Focus then moves to the heading, and they typically proceed to navigate everything that follows that heading.

The line of text indicating distance comes before the heading --- meaning it's likely to be missed. Since distance is likely to be an important piece of information for deciding which rep to choose, this makes it difficult to complete the task.

## Recommended action
Recommendation here is to associate the distance with the heading using `aria-describedby`. Something like:
```
<div class="representative-result-card-content">
	<div class="representative-info-heading">
		<div id="some-unique-id" class="vads-u-font-weight--bold vads-u-font-family--serif">6.16 Mi</div>
		<div class="vads-u-font-family--serif vads-u-margin-top--2p5" id="result-8215">
			<h3 aria-describedby="some-unique-id">Raymond Toczek</h3>
		</div>
		<p style="margin-top: 0px;">American Legion</p>
	</div>
	...
</div>
```

The `aria-describedby` ensures that the distance gets announced alongside the heading, eg. "Heading level 3, Raymond Toczek, 6.16 Mi."

## References

- **WCAG Success Criteria:** N/A
- **Modality:** Screen reader
- **Design System Component:** N/A
- **Design System Pattern:** N/A
- **Content Style Guide:** N/A
- **Context:** N/A
- **Accessibility Defect Severity:** [2: High. Should be fixed in 1-2 sprints post-launch.](https://depo-platform-documentation.scrollhelp.site/developer-docs/accessibility-defect-severity-rubric#Accessibilitydefectseverityrubric-a11y-defect-2)

---

### Next Steps for the VFS Team

- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members` with any questions or to get help validating the issue.
- [ ] **Ticket ownership:** Platform OCTO leadership asks that we document all issues we identify. If you believe that this issue is out of scope or not your team's responsibility, it is still your responsibility to resolve the issue. Please comment on Slack in your team channel tagging `@platform-governance-team-members`, providing an explanation and who you believe is responsible. Governance team will research and follow up.
- [ ] **Close this ticket** when the issue has been resolved or validated by your Product Owner.
